frontend/lib/evictionfree/declaration-builder/confirmation.tsx

```diff
@@ -1,17 +1,151 @@
-import React from "react";
+import React, { useContext } from "react";
+import { OutboundLink } from "../../analytics/google-analytics";
 import { ProgressStepProps } from "../../progress/progress-step-route";
+import { USPS_TRACKING_URL_PREFIX } from "../../../../common-data/loc.json";
 import Page from "../../ui/page";
+import { AppContext } from "../../app-context";
+
+// TO DO: Replace this tracking number with the user's actual one
+const SAMPLE_USPS_TRACKING_NUMBER = "129837127326123";
+
+const HCA_HOTLINE_PHONE_LINK = "tel:+12129624795";
+const NYC_311_CONTACT_LINK =
+  "https://portal.311.nyc.gov/article/?kanumber=KA-02498";
+
+const LIST_OF_ORGANIZING_GROUPS_URL =
+  "https://d3n8a8pro7vhmx.cloudfront.net/righttocounselnyc/pages/1232/attachments/original/1590279936/List_of_Tenant_Organizing_Groups_Across_NY_State.pdf?1590279936";
+
+const checkCircleSvg = require("../../svg/check-circle-solid.svg") as JSX.Element;
+
+const TitleWithSymbol = () => (
+  <div className="media">
+    <div className="media-left">
+      <i className="has-text-info">{checkCircleSvg}</i>
+    </div>
+    <div className="media-content">
+      <h2 className="title is-size-4-mobile">
+        You've sent your hardship declaration
+      </h2>
+    </div>
+  </div>
+);
+
+const RetaliationBlurb = () => (
+  <>
+    <h2 className="title is-spaced has-text-info">
+      Contact a lawyer if your landlord retaliates
+    </h2>
+    <p>
+      It’s possible that your landlord will retaliate once they’ve received your
+      letter. This is illegal. Contact the City's Tenant Helpline (which can
+      provide free advice and legal counsel to tenants) by{" "}
+      <OutboundLink
+        href={`${NYC_311_CONTACT_LINK}`}
```

Comment:
I think this could be easier to read as just `href={NYC_311_CONTACT_LINK}`, unless you strongly feel that template string will consist of just the variable in the near future.